lib/generatePrint.js

```diff
@@ -0,0 +1,47 @@
+const kebab = require('kebab-case')
+const classPostfix = require('postcss-class-postfix')
+const postcss = require('postcss')
+
+const generators = require('./generators')
+
+const RELEVANT_PRINT_GENERATORS = [
+  'spacing',
+  'clears',
+  'display',
+  'flexbox',
+  'floats',
+  'fontStyle',
+  'sizes',
+  'letterSpacing',
+  'lineHeight',
+  'maxWidths',
+  'textDecoration',
+  'textAlign',
+  'textTransform',
+  'typeScale',
+  'typography',
+  'fontWeight',
+  'visibility',
+  'whiteSpace',
+  'widths',
+  // add the @page specific generator
+  'page'
+]
+
+const requirePartial = name => {
+  const kebabed = kebab(name)
+  return require(`../partials/_${kebabed}.css`)
```

Comment:
didn't know this was synchronous (makes sense, just never thought about it)

Reply:
Yeah, it's not great, but I just adapted it from the `generate.js` file. I can refactor at another moment if wanted 🤔

Reply:
no problem, just thought it was a bit weird, but no big deal 🤷‍♂